db/schema.rb

```diff
@@ -299,7 +299,7 @@
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.bigint "fund_id"
-    t.integer "amount"
+    t.decimal "amount", precision: 8, scale: 2
```

Comment:
when i was a wee youth i was raised to never represent currency as a float, lest you get floating point errors in your math (instead using integer cents). are we now a sufficiently advanced society that float currency is ok? is that the idea of the `precision`/`scale`?

Reply:
yeah! There is mercifully a difference between floats and decimals (also called numerics) in postgres. Decimals do not do any of the unholy stuff that floats are notorious for.  https://www.postgresql.org/docs/9.5/datatype-numeric.html#DATATYPE-NUMERIC-DECIMAL

(Though honestly, there's a `money` datatype too, I'll screw around with that a bit in a commit.)

Anyway, it's not that we're an advanced enough society that float currency is okay, just that we're an advanced enough society that we no longer use float most of the time (I think).